### ISSUE TITLE
Change in Video_player for Android from 0.5.4 to 0.6.1 breaks a functionality of my app on a few phones

### ISSUE THREAD

**Issue by Jasonthefirst** [flutter/flutter#18749]
Hi, 

I use the video_player plugin for an app for physical education teachers. In the App teachers can scroll through gymnastic videos and see how their students should execute a physical exercise. To see the differences they are able to scroll frame by frame through the videos. This worked really well until 0.5.4 changed to 0.6.0 and the player for android changed from media player to exo player. I can't seek fast from frame to frame on some devices. It seems like the player only seeks to a few frames but not every single frame. This happens for example on a nexus 7 2013 but not in the local emulator. 

My app opens local files and streams online files if the local file is not yet loaded as well. 

The app opens the file: 

```dart
    videoController = new VideoPlayerController.file(videoFile);
    videoController.addListener(listener);

    videoController.setVolume(0.0);
    videoController.setLooping(true);

    videoController.initialize();
    videoController.play();
```

And while using a Slider the videoController seeks to a corresponding position in the video: 

```dart
    videoController.pause();

    Duration seekpos = new Duration(
    minutes: minutes, seconds: seconds, milliseconds: msseconds);
    videoController.seekTo(seekpos);
```

In the emulator it is working as before but on the Nexus 7 it is juddering a lot. 

```
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel dev, v0.5.5, on Microsoft Windows [Version 10.0.17134.112], locale de-DE)
[√] Android toolchain - develop for Android devices (Android SDK 27.0.3)
[√] Android Studio (version 3.1)
[√] IntelliJ IDEA Ultimate Edition (version 2018.1)
[√] Connected devices (3 available)

• No issues found!
```
Edit: 
The videos are encoded with H264 mp4 with a forced keyint of 1 for smooth scrolling.

**Comment by zoechi** [comment]
flutter/plugins#622 looks like it could fix that

**Comment by cbenhagen** [comment]
Unfortunately not. Frame accurate seek is the default on ExoPlayer. https://github.com/flutter/plugins/pull/622 only adds the same behaviour to the iOS where the default would otherwise be to seek to the next key frame.

**Comment by Jasonthefirst** [comment]
Generally I have a problem, that BOTH versions (mediaplayer and exoplayer) don't play on some Samsung devices (S7 especially). The video starts but only skips between the first two frames. Should I open another Issue or is this related?

**Comment by Jasonthefirst** [comment]
Tried the new update (0.6.4) and it seems a little bit better but only if I slow and wait on a position for a while. I use a slider to seek to a position in the video and with 0.5.4 it works like a charm. But with 0.6.0 and 0.6.4 I can go to the position with the slider and have to wait for a few seconds to resume sliding.

**Comment by collinjackson** [comment]
Here's the commit that switched over to Exoplayer. https://github.com/flutter/plugins/commit/eb17154528bff41a2352ddcf307a97ef7841cd6a in June 2018.

**Comment by collinjackson** [comment]
It's not clear to me what the solution is here but patches are welcome. We don't currently have an active maintainer for this plugin who can help at this time.

@Jasonthefirst feel free to open a new issue for the Samsung device issue.

**Comment by iskakaushik** [comment]
Hey @Jasonthefirst are you still having problems with the video player plugin? Seems like there is an issue with seeking to a position, are you able to see it on the latest version?

**Comment by Jasonthefirst** [comment]
I am really sorry it took me so long to response.
Sadly I still have this problem with the video_player plugin. I still have to use version 0.5.4 because I can't use frame to frame seeking with everything newer. As soon as I switch to the most recent plugin version (0.10.5+2) I can't scroll smoothly through the video. 
As soon as I move the slider a little bit faster the video won't update its frame until I stop moving the slider or move it really slow.

**Comment by TahaTesser** [comment]
Hi @Jasonthefirst 
Could you create a new issue for problems you are facing latest version of `video_player` in detail and explain the reason why you still need to use older version?
Thank you

**Comment by iapicca** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. We are therefore reluctantly going to close this bug for now. Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact descriptions what happens, logs and the output of 'flutter doctor -v'  please.
All system setups can be slightly different so it's always better to open new issues and reference related issues.
